Modules/Core/Common/test/itkHashTableTest.cxx

```diff
@@ -79,19 +79,12 @@ itkHashTableTest(int, char *[])
   lookup(Set, "apple");
   lookup(Set, "durian");
 
-  Set.begin();
```

Comment:
We could remove this entire file, since we have removed `itk::HashTable` in favor of `std::unordered_map`.

Reply:
Yes, but it does not hurt to keep it around.